### ISSUE TITLE
#1 Reunião

### ISSUE THREAD

**Issue by ghost** [training-center/go-study-groups#5]
Salve pessoal, criei a issue para definirmos como começaremos os estudos, tópico esta aberto para sugestões de trilhas, conteúdo, vídeos.. qualquer coisa. 

Como discutido lá no slack #go-studies foi sugerido fazermos a reunião por issues, como ninguém discordou ou teve outra ideia, aqui estamos. Se vocês preferirem podemos mudar nas próximas reuniões. 

Na minha opinião enquanto não precisarmos de vídeo ou áudio por aqui nos serve, pois já fica registrado o que cada um falou e fica fácil também para revisões no futuro.

**Comment by ghost** [comment]
Seguindo modelos dos outros grupos, eu acho que podemos definir desafios por níveis, o que acham? Por exemplo, vejo muitos grupos fazendo calculadoras IMC ou coisa do tipo como desafios iniciais, acho que é uma boa forma de começar praticando por desafios. 

Aqui no nosso repositório já tem bastante material de estudo mas podemos definir **por quais** quem está começando do zero deveria começar.

**Comment by ghost** [comment]
Eae pessoal,

Creio que inicialmente nós poderíamos definir uma **ementa** ou um **roadmap** para que nem se perca, e exercícios.

**Comment by larien** [comment]
Concordo com a ideia de criar um roadmap para estudo. Podemos dividi-lo em niveis, como o Bruno sugeriu, e até acho legal definir vários módulos dentro de cada nível. Então, em cada módulo teríamos esse desafio para consolidar o conhecimento. Algo no esquema freeCodeCamp, Alura, sabem? Vou pensar em algo assim e logo mais lanço uma proposta para vocês xD

**Comment by Diego-Brocanelli** [comment]
Olá, pessoal.

Podemos encerrar essa issue, ou ainda se mantém relevante para permanecer aberta?

**Comment by Diego-Brocanelli** [comment]
Olá, pessoal.

Podemos encerrar essa issue, ou ainda se mantém relevante para permanecer aberta?

**Comment by Diego-Brocanelli** [comment]
Olá, pessoal.

Podemos encerrar essa issue, ou ainda se mantém relevante para permanecer aberta?
